packages/generator/templates/app/src/app/auth/forgot-password/page.tsx

```diff
@@ -0,0 +1,46 @@
+"use client"
+import {LabeledTextField} from "src/app/components/LabeledTextField"
+import {Form, FORM_ERROR} from "src/app/components/Form"
+import {ForgotPassword} from "../validations"
+import forgotPassword from "../mutations/forgotPassword"
+import {useMutation} from "@blitzjs/rpc"
+import {BlitzPage} from "@blitzjs/next"
+
+const ForgotPasswordPage: BlitzPage = () => {
+  const [forgotPasswordMutation, {isSuccess}] = useMutation(forgotPassword)
```

Comment:
Based on what I learned today about how to add the metadata API to NextJS-App-Directory, I think the starter should make all pages server components and add the metadata to them. Which means extracting the forms in `(auth)` into separate `(auth)/foo/_components/FooForm.tsx` client component files.

See also https://github.com/blitz-js/blitzjs.com/pull/851/files